sim/druid/talents.go

```diff
@@ -106,6 +93,63 @@ func (druid *Druid) ApplyTalents() {
 	druid.applyPrimalMadness()
 }
 
+func (druid *Druid) applyBalanceOfPower() {
+	if druid.Talents.BalanceOfPower > 0 {
+		druid.AddStaticMod(core.SpellModConfig{
+			School:     core.SpellSchoolArcane | core.SpellSchoolNature,
+			ClassMask:  DruidArcaneSpells | DruidNatureSpells,
+			FloatValue: 0.01 * float64(druid.Talents.BalanceOfPower),
+			Kind:       core.SpellMod_DamageDone_Pct,
```

Comment:
most bonus damage talents in wow are additive which is `core.SpellMod_DamageDone_Flat`

Reply:
I got the information off the collective mind in the druid classic Discord :)